src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -8518,9 +8523,6 @@ extern "C" bool QCALLTYPE SfiNext(StackFrameIterator* pThis, uint* uExCollideCla
         }
         else
         {
-            // TODO-NewEH: Currently there are two other cases of internal VM->managed transitions. The FastCallFinalize and COMToCLRDispatchHelperWithStack
```

Comment:
We don't propagate managed exceptions through the `COMToCLRDispatchHelper`, they are caught and transformed into HRESULT, so after the FastCallFinalize was removed by another PR, this comment is now obsolete.